cleansweep/models.py

```diff
@@ -370,6 +373,39 @@ def get_stats(self, name, limit=100, total=False):
             .limit(limit))
         return q.all()
 
+    def get_stats_member_daily(self):
+        q = (db.session.query(Member.id)
+            .filter(
+                place_parents.c.parent_id==self.id,
+                place_parents.c.child_id==Member.place_id,
+                func.to_char(Member.created,'dd/mm/yyyy') == func.to_char(func.current_date(),'dd/mm/yyyy')))
+        return q.count()
+
+    def get_stats_member_yesterday(self):
+        q = (db.session.query(Member.id)
+            .filter(
+                place_parents.c.parent_id==self.id,
+                place_parents.c.child_id==Member.place_id,
+                func.to_char(Member.created,'dd/mm/yyyy') == 
+                func.to_char(func.current_date()-1,'dd/mm/yyyy')))
+        return q.count()
+
+    def get_stats_member_weekly(self):
+        q = (db.session.query(Member.id)
+            .filter(
+                place_parents.c.parent_id==self.id,
+                place_parents.c.child_id==Member.place_id,
+                func.to_char(Member.created,'dd/mm/yyyy') > func.to_char(func.date_trunc('week',func.current_date()),'dd/mm/yyyy')))
+        return q.count()
+
+    def get_stats_member_monthly(self):
+        q = (db.session.query(Member.id)
+            .filter(
+                place_parents.c.parent_id==self.id,
+                place_parents.c.child_id==Member.place_id,
+                func.to_char(Member.created,'dd/mm/yyyy') > func.to_char(func.date_trunc('month',func.current_date()),'dd/mm/yyyy')))
+        return q.count()
+
```

Comment:
Using `to_char` is very inefficient, esp. in postgres. I think the easiest way to group by date for all the recent dates and sum it up in Python.

The query to use in Postgres would be something like this:

    select created::date, count(*) from member where CURRENT_DATE - created::date < 28 group by 1;